cypress_shared/fixtures/offences-dev.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
```

Comment:
Do we need to test with a certain CRN for these to work or are they more general than that?